tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/tuples/TupleAccessExprNegativeTest.java

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
We have `org.ballerinalang.test.expressions.access.MemberAccessTest` already. Even if we need a separate test file IMO this should go under `org.ballerinalang.test.expressions.access` rather than tuples.

Reply:
I think we don't need a separate file for this. I will move tests to `MemberAccessTest` class.

Reply:
Updated. 1ef87477be4270f73cec77393ed6e541fc0886eb